src/main.py

```diff
@@ -1,21 +1,52 @@
 import os
+import sentry_sdk
 import whisper
 import logging
 import tempfile
 from datetime import datetime
 import urllib.parse
 from flask import Flask
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.rq import RqIntegration
 from flask import request
 from flask import render_template, Response
+import redis
 from rq import Queue
 from rq.job import Job
 from rq.exceptions import NoSuchJobError
 
 from src.utils import generate_srt, generate_vtt, generate_text
-from src.worker import conn
 from src import mailer
 
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+
+if SENTRY_DSN:
+    print("Sentry detected, Using " + SENTRY_DSN)
```

Comment:
Should this be print?

Reply:
Yes. Because It can't log using the `app.logger.info()` because sentry needs to be initialised before the everything else :-)